File: Database/test/TestHarness.java


public class TestHarness
{

	/**
	 * Tests all the unit tests together.
	 * To add a test, ignore the make file, just put the call here.
	 */

	public static void main ( String [] args )
	{
		TreeTest.run();
		StarTest.run();

		System.out.println("\n\n" + UnitTest.testsPassed + " of " + UnitTest.testNum + " passed\n\n");
	}



}

File: Star Tracker/test/TestHarness.cpp
#include "UnitTest.hpp"

int main ()
{
	int numTests = 0;
	int testsPassed = 0;


	testsPassed += stT::run();
	numTests++;

	testsPassed += ipT::run();
	numTests++;


	cout << endl << endl
		 << testsPassed << " of " << numTests << " passed" << endl << endl;
	return 0;
}
